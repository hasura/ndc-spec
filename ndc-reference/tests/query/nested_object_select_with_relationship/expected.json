[
  {
    "rows": [
      {
        "name": "Queen Mary University of London",
        "staff": [
          {
            "first_name": "Peter",
            "last_name": "Landin",
            "author": {
              "rows": [
                {
                  "id": 1,
                  "first_name": "Peter",
                  "last_name": "Landin"
                }
              ]
            }
          }
        ]
      },
      {
        "name": "Chalmers University of Technology",
        "staff": [
          {
            "first_name": "John",
            "last_name": "Hughes",
            "author": {
              "rows": [
                {
                  "id": 2,
                  "first_name": "John",
                  "last_name": "Hughes"
                }
              ]
            }
          },
          {
            "first_name": "Koen",
            "last_name": "Claessen",
            "author": {
              "rows": []
            }
          }
        ]
      },
      {
        "name": "Stockholm University",
        "staff": [
          {
            "first_name": "Jelena",
            "last_name": "Zdravkovic",
            "author": {
              "rows": []
            }
          }
        ]
      }
    ]
  }
]