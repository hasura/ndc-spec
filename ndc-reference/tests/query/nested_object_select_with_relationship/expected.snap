---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/nested_object_select_with_relationship/request.json
---
[
  {
    "rows": [
      {
        "name": "Queen Mary University of London",
        "staff": [
          {
            "first_name": "Peter",
            "last_name": "Landin",
            "author": {
              "rows": [
                {
                  "id": 1,
                  "first_name": "Peter",
                  "last_name": "Landin"
                }
              ]
            }
          }
        ]
      },
      {
        "name": "Chalmers University of Technology",
        "staff": [
          {
            "first_name": "John",
            "last_name": "Hughes",
            "author": {
              "rows": [
                {
                  "id": 2,
                  "first_name": "John",
                  "last_name": "Hughes"
                }
              ]
            }
          },
          {
            "first_name": "Koen",
            "last_name": "Claessen",
            "author": {
              "rows": []
            }
          }
        ]
      },
      {
        "name": "Stockholm University",
        "staff": [
          {
            "first_name": "Jelena",
            "last_name": "Zdravkovic",
            "author": {
              "rows": []
            }
          }
        ]
      }
    ]
  }
]
